Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -441,10 +465,12 @@ static std::optional<SRGBA<uint8_t>> finishParsingHexColor(uint32_t value, unsig
     return std::nullopt;
 }
 
-template<typename CharacterType> static std::optional<SRGBA<uint8_t>> parseHexColorInternal(std::span<const CharacterType> characters)
+template<typename CharacterType>
+static std::optional<SRGBA<uint8_t>> parseHexColorInternal(std::span<const CharacterType> characters)
```

Comment:
@darinadler is pretty fond of having the `template<>` part all on the same line. 

I don't have strong opinions, but rather than ping pong these lines, we should probably standardize on a style for this.

Reply:
I prefer having `template<>` on its own line because it leave the next line looking mostly like a normal function.